src/content/tutorials/observability/multi-tenancy/_index.md

```diff
@@ -1,15 +1,201 @@
+
+## Understanding multi-tenancy
+
+Multi-tenancy works through three key components:
+
+### Tenants
+
+A **tenant** is a logical namespace that isolates observability data (metrics, logs) in the backend storage systems (Mimir and Loki). When applications send data, they include a tenant label that determines where the information goes.
+
+**Key characteristics:**
+
+- Data isolation at the storage level
+- Independent retention policies and limits
+- Separate query scope for dashboards and alerts
+
+### Grafana organizations
+
+A **Grafana organization** gives you access to specific tenants through dedicated datasources and dashboards. Each organization is a separate workspace in Grafana with its own:
+
+- User roles and permissions
+- Dashboards and folders
+- Datasource configurations
+- Alert rules and notification channels
+
+### RBAC groups
+
+**RBAC groups** from your identity provider (like Active Directory or OAuth) define which users get access to specific Grafana organizations and their permission levels.
+
+## Tenant separation strategies
+
+Choose the right strategy based on your needs:
+
+### By environment
+
+Separate tenants by deployment environment:
+
+```yaml
+tenants:
+- production
+- staging  
+- development
+```
+
+**Use case:** Keep dev/staging data separate from production, apply different retention policies.
+
+### By team or department
+
+Organize tenants around teams:
+
+```yaml
+tenants:
+- platform-team
+- frontend-team
+- backend-team
+- data-team
+```
+
+**Use case:** Give teams ownership of their data, enable self-service monitoring.
+
+### By service or application
+
+Create separation at the service level:
+
+```yaml
+tenants:
+- user-service
+- payment-service
+- notification-service
+- analytics-service
+```
+
+**Use case:** Microservices where each service team manages their own monitoring.
+
+### Hybrid approach
+
+Combine strategies using consistent naming:
+
+```yaml
+tenants:
+- prod-frontend
+- prod-backend
+- staging-frontend
+- staging-backend
+```
+
+**Use case:** Balance granular control with manageable complexity.
+
+## Tenant naming best practices
+
+Follow these guidelines when choosing tenant names:
+
+### Naming requirements
+
+Tenant names must follow [Grafana Mimir tenant ID restrictions](https://grafana.com/docs/mimir/latest/configure/about-tenant-ids/):
+
+- Alphanumeric characters (a-z, A-Z, 0-9) and special characters (!, -, _, ., *, ', (, ))
+- Between 1 and 150 characters
+- Can't use: `.`, `..`, `__mimir_cluster`
+
+### Recommended patterns
+
+**Environment-based:** `production`, `staging`, `development`
+
+**Team-based:** `platform-team`, `frontend-team`, `backend-team`
+
+**Service-based:** `user-service`, `payment-service`, `notification-service`
+
+**Hybrid:** `prod-frontend`, `stage-backend`, `dev-notifications`
+
+## Access control patterns
```

Comment:
This section is a duplicate and less informative version of `Creating a Grafana organization` section, I think this should be removed here as this doc is only about the how things works and not how to do it.
How to name tenant was already covered in the previous section.